index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>temp</title>
+<script src = "temp.js"></script>
```

Comment:
@b232wang: Could you use more self-explanatory names like `animalcules.js` than `temp.js`?